lib/raygun.ts

```diff
@@ -24,8 +25,9 @@ import {
   Tag,
   Transport,
 } from "./types";
+import * as breadcrumbs from "./breadcrumbs";
```

Comment:
This is deviating from the more commonly used `raygun.<something>.ts` scheme.

Reply:
renamed to `raygun.breadcrumbs.ts`